optional-operators-subscribe.yml

```diff
@@ -2,7 +2,7 @@
 - name: "Run optional operators subscribe"
   hosts: localhost
   become: false
-  gather_facts: false
+  gather_facts: true
```

Comment:
Just a quick point - depending on the DCI's environment, gathering facts might fail for them. 
@14rcole do you think if this is a legitimate worry on my part?

Reply:
@dirgim Gather facts should work with any env since it takes variables related to the system. 
Why do you think it doesnt work ?

Reply:
I've seen it fail in some weird environments in the past. But I think it should work fine in DCI environments since they use Ansible for other tasks too, closing this discussion.